core/client/fs/pom.xml

```diff
@@ -98,6 +98,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.alluxio</groupId>
+          <artifactId>alluxio-underfs-local</artifactId>
+          <version>${project.version}</version>
+          <scope>test</scope>
+      </dependency>
```

Comment:
please keep the indentation aligned consistently

Reply:
Reformatted.